src/test/kotlin/no/nav/syfo/testhelper/ExternalMockEnvironment.kt

```diff
@@ -10,17 +10,23 @@ class ExternalMockEnvironment() {
     val eregMock = EregMock()
     val stsMock = STSMock()
     val dokDistMock = DokDistMock()
+    val fastlegeMock = FastlegeMock()
+    val adresseregisterMock = AdresseregisterMock()
```

Comment:
Dissen mockene er basert på at det er server som sender responser som er http/json. Tror du må lage noe eget for å mocke WS eller evt. bruke mockk.

Reply:
Ser mer på dette når jeg forsøker å lage tester for Soap-tjenestene (i en egen PR).